### PR TITLE
DateTime formatting

### PR DESCRIPTION
**Pull request by mediumTaj**
### Summary
To be merged after https://github.com/watson-developer-cloud/unity-sdk/pull/539

This pull request correctly formats datetime and updates the Unity SDK readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,23 +149,18 @@ void TokenExample()
...
-    Log.Debug("OnFail()", "Failed: {0}", error.ToString());
+    Log.Debug("OnListWorkspaces()", "Response: {0}", response.Response);
 }
 ```
 
 ### Username and password
 ```cs
-using IBM.Watson.DeveloperCloud.Services.Assistant.v1;
+using IBM.Watson.Assistant.v1;
 using IBM.Watson.DeveloperCloud.Utilities;
```
Comment (mamoonraja) [inline README.md]: Are utilities still in `DeveloperCloud`?

File: README.md [hunk 2]
```diff
@@ -180,37 +171,25 @@ void Start()
 ```
 
 ## Callbacks
-Success and failure callbacks are required. You can specify the return type in the callback.  
+Success callbacks are required. You can specify the return type in the callback.
```
Comment (mamoonraja) [inline README.md]: I like it how failure callbacks are handled by `IBMError` now 👍

File: README.md (hunk 1 above)
Comment (mediumTaj) [reply README.md]: You're right! should be `IBM.Cloud.SDK.Utilities`

**Pull request merged**